test/gie/builtins.gie

```diff
@@ -1423,9 +1423,9 @@ expect  -0.001790143 -0.000895247
 ===============================================================================
 
 -------------------------------------------------------------------------------
-operation +proj=etmerc   +ellps=GRS80 +zone=30
+operation +proj=etmerc   +ellps=GRS80
 -------------------------------------------------------------------------------
-tolerance 0.1 mm
+tolerance 50e-9 m
```

Comment:
```suggestion
tolerance 50 nm
```
works as well and is probably easier to read. Can be applied in other places as well.

Reply:
I didn't see "nm" listed as an allowed unit with `proj -lu`?

Reply:
That's true, but actually gie doesn't use the units from `proj -lu`, instead it define its own set of units : 
https://github.com/OSGeo/PROJ/blob/91a767b0cdebba317e20abdcee6bfa444cfe8455/src/apps/gie.cpp#L460-L496

I realise that the documentation is wrong about this.

Reply:
Fixed